.github/dependabot.yml

```diff
@@ -9,6 +9,11 @@ updates:
     labels:
       - "ok-to-test"
       - "release-note-none"
+    groups:
+      kubernetes:
+        patterns:
+          - "k8s.io/*"
+          - "sigs.k8s.io/*"
     ignore:
```

Comment:
Should we also update the ignore section? I'm not very familiar with dependabot, but maybe we at least need to have the same policy for `sigs.ks8.io` ...

cc: @alculquicondor

Reply:
Looking at this, do we see any value in having all sig projects updated together? I see that this repo tends to have very different projects in the sig group so I think individual updates is fine. 

Ignore is mostly about doing updates via patches only.